pkg/plugin/v2/scaffolds/init.go

```diff
@@ -40,6 +40,9 @@ const (
 	ControllerToolsVersion = "v0.3.0"
 	// KustomizeVersion is the kubernetes-sigs/kustomize version to be used in the project
 	KustomizeVersion = "v3.5.4"
+	// ControllerRuntimeEnvTestVersion version to be used to download the envtest setup script
+	// todo: update the tag release when the next version of the project be released with this script
+	ControllerRuntimeEnvTestVersion = "master"
```

Comment:
We almost certainly shouldn't be pinning at `master` -- this is gonna break

Reply:
we may also just want to have a separate script of kubebuilder or something that we scaffold out, or a go module that we can `go get` that does the downloading.

Reply:
e.g. have a separate module in `kubebuilder/tooling` (so it has its own deps), and then we can `go get`.

Reply:
I really liked the idea of the tooling and I also agree that shows better we do not need to create a dependency to controller runtime to just download the bins.